packages/inspector/package.json

```diff
@@ -45,6 +45,7 @@
     "@unocss/core": "workspace:*",
     "@unocss/rule-utils": "workspace:*",
     "gzip-size": "catalog:",
+    "kolorist": "^1.8.0",
```

Comment:
We have `colorette` dependency in pnpm workspace, you can use it with `catalog` rather than install a new dependency.
https://github.com/unocss/unocss/blob/d32841ce9e1ada8b1fd81d980daba5d2c6cf9ab1/pnpm-workspace.yaml#L75

Reply:
@zyyv Great catch, thank you!